tox.ini

```diff
@@ -2,5 +2,5 @@
 max-line-length=120
 application_import_names=pysite
 ignore=P102,B311,W503,E226,S311
-exclude=__pycache__, venv, app_test.py, .venv
+exclude=__pycache__, venv, .venv, tests
```

Comment:
this is flake8, right? why do we exclude tests from linting?

Reply:
we exclude tests from linting because some tests require some terrible python code that flake8 really dislikes